httpie/utils.py

```diff
@@ -96,16 +99,17 @@ def is_expired(expires: Optional[float]) -> bool:
         return expires is not None and expires <= now
 
     attr_sets: List[Tuple[str, str]] = parse_ns_headers(
-        value for name, value in headers
-        if name.lower() == 'set-cookie'
+        RE_COOKIE_SPLIT.split(cookies)
     )
+
     cookies = [
         # The first attr name is the cookie name.
         dict(attrs[1:], name=attrs[0][0])
         for attrs in attr_sets
     ]
 
     _max_age_to_expires(cookies=cookies, now=now)
+    "Set-Cookie: foo=bar; Expires=Mon, 21 Jun 2021 04:20:21 GMT; Max-Age=1; Path=/"
```

Comment:
Is it a comment left for debugging or to expose the cookie syntax?

Reply:
Note to self that I forgot to delete.